### PR TITLE
opt add-json-iterator-with-build-tags

### PR DESCRIPTION
**Pull request by Chyroc**

### CODE REVIEW

**Comment by Chyroc** [comment]
@VojtechVitek  More humane and beautiful way to add json-iterator by `go build -tags=jsoniter .`, and then I will close https://github.com/go-chi/render/pull/9, I hope to pass the review, thank you

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ script:
   - go get -d -t ./...
   - go test ./...
   - >
-    goimports -d -e ./ | grep '.*' && { echo; echo "Aborting due to non-empty goimports output."; exit 1; } || :
```
Comment (Chyroc) [inline .travis.yml]: If don't have this, then `goimports ` will think it doesn't need this sentence: `import "github.com/json-iterator/go"`.

and failed ci: https://travis-ci.org/go-chi/render/jobs/334587235#L479-L494

File: .travis.yml [hunk 2]
```diff
@@ -12,4 +12,4 @@ script:
...
+    goimports -d -e $(find ./* -type f -name '*.go' | grep -v 'jsoniter') | grep '.*' && { echo; echo "Aborting due to non-empty goimports output."; exit 1; } || :
```
Comment (Chyroc) [inline .travis.yml]: If don't have this, then goimports will think it doesn't need this sentence: import "github.com/json-iterator/go".

and failed ci: https://travis-ci.org/go-chi/render/jobs/334587235#L479-L494

**Comment by Chyroc** [comment]
Thank you very much for your approval. I tested this change in my own project (via the `init` function in different files) and the result is correct. I do not know what happened to your test result. If there are any problems, please correct me.

**Comment by VojtechVitek** [comment]
@Chyroc I haven't had chance to test this out yet. But I will. 👍 
I'm very busy recently, so please be patient, it might take some time.

**Pull request closed without merging**